aws/alarms/lambda.tf

```diff
@@ -3,7 +3,7 @@
 #
 data "archive_file" "notify_slack" {
   type        = "zip"
-  source_file = "lambda/notify_slack/notify_slack.js"
+  source_file = "lambda/notify_slack/notify_slack.ts"
```

Comment:
Since we are now converting the lambda to Typescript, you will have to bundle the build folder resulting from the `yarn build` command. 
It should produce a `dist` folder where you will have a `notify_slack.js` file that is your entry point.

Here is an example of how it was done for other lambdas:
```
data "archive_file" "audit_logs_code" {
  type        = "zip"
  source_dir  = "./code/audit_logs/dist"
  output_path = "/tmp/audit_logs_code.zip"
}
```